src/workloads/scale/ManyUpdate.yml

```diff
@@ -0,0 +1,72 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-replication"
+Description: |
+  This workload loads a large number of small documents into a single collection, and then does a
+  small multi-update which touches all of them.  It is intended to measure the impact of the load of
+  replication of many oplog entries on the primary.  Number of documents should be significantly
+  greater than the maximum replication batch size of 5K (50K is a good minimum).
```

Comment:
Is there a chance flow control will kick in on this workload due to the w:1 below? Then we are measuring flow control behavior rather than replication.

Reply:
Sounds great. thanks!